nexus/src/context.rs

```diff
@@ -111,12 +111,22 @@ impl ServerContext {
             .register_producer(external_latencies.clone())
             .unwrap();
 
-        let assets_directory = env::var("CARGO_MANIFEST_DIR")
-            .map(|root| {
-                PathBuf::from(root)
-                    .join(config.console.assets_directory.to_owned())
-            })
-            .ok();
+        // Support both absolute and relative paths. If configured dir is
+        // absolute, use it directly. If not, assume it's relative to the
+        // current working directory.
+        let static_dir = if config.console.static_dir.is_absolute() {
+            Some(config.console.static_dir.to_owned())
+        } else {
+            env::current_dir()
+                .map(|root| root.join(config.console.static_dir.to_owned()))
+                .ok()
+        };
+
+        // we don't want to fail outright yet, but we do want to try to make
+        // problems slightly easier to debug
+        if static_dir.is_none() {
+            println!("WARNING: no assets directory configured. All console page and asset requests will 404.");
+        }
```

Comment:
🤷

Reply:
Error level log message?

Reply:
Updating. it's unlikely this will happen anyway because the only way is if the `current_dir()` errors